test/config/utility.cc

```diff
@@ -0,0 +1,96 @@
+#include "test/config/utility.h"
+
+#include "common/common/assert.h"
+#include "common/protobuf/utility.h"
+
+#include "test/test_common/environment.h"
+#include "test/test_common/network_utility.h"
+
+#include "api/filter/http_connection_manager.pb.h"
+
+namespace Envoy {
+
+std::string basic_config = R"EOF(
+static_resources:
+  listeners:
+    name: listener_0
+    address:
+      socket_address:
+        address: 127.0.0.1
+        port_value: 0
+    filter_chains:
+      filters:
+        name: http_connection_manager
+        config:
+          httpFilters:
```

Comment:
nit: Can we make all of these consistently snake_case?

Reply:
Apparently we can.  Fascinating.
(was just editing what proto.DebugString() spat out)